### PR TITLE
Adding execution app in katana

### PR DESCRIPTION
**Pull request by apathapa-warrior**

### CODE REVIEW

File: katana/config.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "engineer": "",
-    "pythonsrcdir": "",
-    "xmldir": "",
-    "testsuitedir": "",
-    "projdir": "",
-    "idfdir": "",
-    "testdata": "",
+    "pythonsrcdir": "/data/users/apathapa/fnc/python3/2129/warriorframework_py3/warrior",
```
Comment (santhosh-warrior) [inline katana/config.json]: why the path is hard-code ?

File: katana/wapps/execution/templates/execution/execution_settings.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"defaults": {"start_dir": "default", "jira_proj": "project3"}, "jira_proj": [{"name": "warrior"}, {"name": "project1"}, {"name": "project2"}, {"name": "project3"}]}
```
Comment (santhosh-warrior) [inline katana/wapps/execution/templates/execution/execution_settings.json]: Why added default Jira project (warrior) is added ?

File: katana/wapps/execution/wf_config.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "icon": "play-circle",
+  "color": "blue",
+  "developer": "Warrior Framework",
+  "description": "Execute Warrior using this App",
```
Comment (santhosh-warrior) [inline katana/wapps/execution/wf_config.json]: Execute Warrior CLI using the App

File: katana/config.json (hunk 1 above)
Comment (apathapa-warrior) [reply katana/config.json]: I have made necessary changes and committed

File: katana/wapps/execution/wf_config.json (hunk 3 above)
Comment (apathapa-warrior) [reply katana/wapps/execution/wf_config.json]: Made necessary changes and committed.

File: katana/wapps/execution/templates/execution/execution_settings.json (hunk 2 above)
Comment (apathapa-warrior) [reply katana/wapps/execution/templates/execution/execution_settings.json]: Made necessary changes and committed.

**Pull request merged**